src/reducers/newQuoteReducer.js

```diff
@@ -0,0 +1,19 @@
+import {NEW_QUOTE} from '../actions';
+
+// initial state
+const initState ={
+    quote:'',
+    author:''
+}
+// create reducer
+const newQuoteReducer=(preState=initState,action)=>{
+    switch(action.type){
+        case NEW_QUOTE:
+            console.log(action)
+            const {quote ,author} = action.quote
+            return { quote,author }
+        default:
+            return preState;
+    }
+}
```

Comment:
Don't mutate the state. You can use the object spread syntax:
`case NEW_QUOTE:
   const { quote, author } = action.quote
   return { ...state, quote, author }`